package.json

```diff
@@ -33,6 +33,7 @@
         "i18next-browser-languagedetector": "^4.0.2",
         "i18next-xhr-backend": "^3.2.2",
         "libphonenumber-js": "^1.7.49",
+        "lodash": "^4.17.15",
```

Comment:
lodash is very heavy, we could probably go without it without major issue don't you think?

Reply:
Yes it was for simplicity. I'll remove it. I wanted to use optional chaining but I don't know if it's already ready. I saw it didn't work on Firefox android

Reply:
maybe CRA pollyfil it?